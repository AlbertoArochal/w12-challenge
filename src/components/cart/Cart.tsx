import { Header } from '../Header/header';
import { FetchCartList } from './FetchCartList';

export const Cart = () => {
    return (
        <>
            <Header />
            <FetchCartList />
        </>
    );
};
