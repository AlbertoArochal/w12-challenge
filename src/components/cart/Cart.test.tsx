import { Cart } from './Cart';
import { render } from '@testing-library/react';

jest.mock('../Header/header', () => ({
    Header: () => <div>Header</div>,
}));

jest.mock('./FetchCartList', () => ({
    FetchCartList: () => <div>FetchCartList</div>,
}));

describe('Cart', () => {
    it('should render', () => {
        const { container } = render(<Cart />);
        expect(container).toMatchSnapshot();
    });
});
