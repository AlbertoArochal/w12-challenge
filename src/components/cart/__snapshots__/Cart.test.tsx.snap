// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart should render 1`] = `
<div>
  <div>
    Header
  </div>
  <div>
    FetchCartList
  </div>
</div>
`;
